//my reddit simulation proto file

syntax  = "proto3";

message Request {
    int64 client_id = 1;
    string request_data = 2;
}

message Response {
    int64 server_id = 1;
    string response_data = 2;
}


service DataStreamingService{
	rpc StartDatastream(Request) returns (stream Response);
}